### DISCUSSION TITLE
Online Version and/or web sharing

### DISCUSSION THREAD

**Discussion by DVegGyver** [phase1geo/Minder#515] [Q&A]
I used many Mental maps application before Minder. All are online servire that I can visualize/edit on my phone.
There are some method to do the same?

for example, exporting in some format and share the file with my phone for visualize the minder map (and eventually edit it)?

**Comment by phase1geo** [comment]
There is no such service that would allow you to edit the mindmap.

**Comment by DVegGyver** [comment]
Ok, so the only way is to export in some compatible format (like mindfree o xmind ) and usa a app that open such format.
Than save and open back the new file with minder.
This is a workaround that I think could work.
